bower.json

```diff
@@ -19,6 +19,6 @@
     "bower_components"
   ],
   "dependencies": {
-    "webcomponentsjs": "^0.7.22"
+    "webcomponentsjs": "^1.0.0"
```

Comment:
Please change `dependencies` to `devDependencies`. 

This is how Polymer elements do this: https://github.com/PolymerElements/paper-dialog-scrollable/blob/852b67551f34f1e48ec51d0e36c38576871e0b3d/bower.json#L35